### DISCUSSION TITLE
What will be the Osu!Lazer iOS App Store and android google play release date

### DISCUSSION THREAD

**Discussion by VortexDash229YT** [ppy/osu#30764] [Q&A]
How what you think is the release date for Osu!Lazer iOS App Store and google play store release i think it will be may 12, 2025 or Early June i don’t known so I have confirmed image
![IMG_3723](https://github.com/user-attachments/assets/ff8405a0-c9ee-4af1-99e3-f5a3ad6842dc)

**Comment by Joehuu** [comment]
There are no exact date estimates because the last time it was done (in the [readme](https://github.com/ppy/osu/blob/master/README.md#latest-release)), it wasn't fulfilled:
> Our goal is to get the game on mobile app stores in early 2024.

It wasn't updated/removed since.

**Comment by Xtranormal-guy** [comment]
2027?

**Comment by Kuina-chan** [comment]
In the lastest [update video](https://www.youtube.com/watch?v=ZS3EucVtfB8), peppy said in the roadmap on screen that he wants to release without making a big deal about it, but we have to see, so I think it can be this year or next year at most

**Comment by Tenexxt** [comment]
I've actually asked the team recently about this and all the necessary preparations are done now, so it might release even today or on the next new year's eve - basically whenever they want to release it.
I assume that before release there still be some more updates targeted to mobile, but that's only my guess. All we can do for now is wait for the release, tho I personally feel it won't be too far away.
